website/public/js/controllers/inventoryCtrl.js

```diff
@@ -152,12 +160,16 @@ habitrpg.controller("InventoryCtrl",
         $scope.selectedFood = null;
 
         _updateDropAnimalCount(user.items);
+        if ($scope.mountCount > startingMounts && !user.preferences.suppressModals.raisePet) {
+          $rootScope.raisedPet = {displayName: petDisplayName, spriteName: pet, egg: egg, potion: potion}
```

Comment:
Same question here, I think we can just insert this into the scope of the modal.